js/src/dapps/tokenreg/Inputs/Text/input-text.js

```diff
@@ -105,22 +105,22 @@ export default class InputText extends Component {
     const { validationType, contract } = this.props;
     const validation = validate(value, validationType, contract);
 
-    if (validation instanceof Promise) {
```

Comment:
It doesn't work reliably with the `es6-promise` shim, which I'm guessing is included in FF. (`Bluebird` has the same issue.) 

Chrome doesn't show it since it has native Promises.